pkg/kubectl/cmd/set/set.go

```diff
@@ -44,6 +44,7 @@ func NewCmdSet(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 
 	// add subcommands
 	cmd.AddCommand(NewCmdImage(f, out))
+	cmd.AddCommand(NewCmdResources(f, out, err))
```

Comment:
Sorry, when I changed the name of the command I forgot to add the changed filename to the git repo.